database/src/lib.rs

```diff
@@ -45,3 +45,21 @@ impl<T> SqlxResultExt<T> for Result<T, sqlx::Error> {
         }
     }
 }
+
+#[cfg(test)]
+mod test_util {
+    use once_cell::sync::Lazy;
+    use sqlx::PgPool;
+    use tokio::sync::Mutex;
+
+    pub async fn pool() -> &'static PgPool {
+        static INITIALIZED: Lazy<Mutex<bool>> = Lazy::new(|| Mutex::new(false));
+        let mut initialized = INITIALIZED.lock().await;
+        if !*initialized {
+            dotenv::dotenv().unwrap();
+            migrations::initialize().await;
+            *initialized = true;
+        }
+        migrations::pool()
```

Comment:
The test is only run once, is there a reason you write code like the test is being run multiple times?

Reply:
I plan on having a lot of database tests, not one. Each test has to have this called, otherwise, it can't run on its own. And if multiple are running, only one should initialize it.